src/frontend/src/store/index.ts

```diff
@@ -299,7 +280,7 @@ export default new Vuex.Store({
             commit("MUTATION_SET_LAST_NAME", response.data.user.last_name)
             commit("MUTATION_SET_UMNETID", umnetId)
             commit("MUTATION_SET_PASSWORD", password)
-            commit("MUTATION_SET_WALLETID", response.data.user.public_key)
+            commit("MUTATION_SET_WALLETID", umnetId)
```

Comment:
Instead of doing this, I'd recommend removing the walletID property all-together and replace every instance across the app with umnetId!

Reply:
same idea here, do we want to abstract the walletIds from umnetIds?

Reply:
if we plan to change walletId to something else in the future, having these actions set up would help. Currently, it was one line change to change walletId.

Reply:
Not sure what you mean by abstract but I think umetId should just replace walletId entirely

Reply:
To clarify: For QoL of the project, having walletIds separately will help. If the walletId was to be changed in the future in the backend, the frontend will only require one line change here. If we remove Actions associated to the walletId completely, we are losing that convenience.

Reply:
done

Reply:
I am still not convinced because it seems redundant to me to have two fields that will always contain the same value no matter what., and even more so when we send both values in a request and they are the same. It doesn't seem to me that in the next month we'll have a need to change from using umnetId to index into the wallet dictionary, as it satisfies our uniqueness requirement.

Reply:
Do you mean by changing walletId entirely to umnetId in the backend as well?